addons/stock_landed_costs/models/stock_landed_cost.py

```diff
@@ -196,15 +196,15 @@ def get_valuation_lines(self):
 
         for move in self._get_targeted_move_ids():
             # it doesn't make sense to make a landed cost for a product that isn't set as being valuated in real time at real cost
-            if move.product_id.cost_method not in ('fifo', 'average') or move.state == 'cancel' or not move.product_qty:
+            if move.product_id.cost_method not in ('fifo', 'average') or move.state == 'cancel' or not move.quantity:
```

Comment:
I didn't saw it the first time but `product_qty` is the quantity in product's uom while `quantity` is expressed in stock move's uom. So we will need to convert it before passing it to `vals`